Resources/Prototypes/Entities/Objects/Specific/atmos.yml

```diff
@@ -20,10 +23,10 @@
   - type: Appearance
   - type: GenericVisualizer
     visuals:
-      enum.GasAnalyzerVisuals.VisualState:
-        analyzer:
-          Off: { state: icon }
-          Working: { state: working }
+        enum.FilterVisuals.Enabled:
```

Comment:
using filtervisuals for this seems strange, just make another enum

Reply:
For some reason it didn't work the first time I tried, but I was definitely doing something wrong. it's got it's own enum now